node.go

```diff
@@ -23,6 +23,7 @@ import (
 	"golang.org/x/sync/errgroup"
 
 	iatomic "github.com/aerospike/aerospike-client-go/v8/internal/atomic"
+	internal "github.com/aerospike/aerospike-client-go/v8/internal/version"
```

Comment:
Just import as `"github.com/aerospike/aerospike-client-go/v8/internal/version"`. No need for a qualifier.